README.md

```diff
@@ -1,52 +1,97 @@
 # Makeflow for Drone Processing Pipeline
 
-## Build the container
+## Running the container
+This section contains different ways of executing an existing container.
 
-```bash
-cp jx-args.json.example jx-args.json
-docker build --progress=plain -t agpipeline/scif-drone-pipeline:1.3 .
+### Terms
+Here are the definition of some of the terms we use with links to additional information
+
+* BETYdb <a name="betydb" />
+[BETYdb](https://www.betydb.org/) is a database that can be used to store trait and yield data.
+It can be used in the processing pipeline as a source of plot geometry for clipping.
+
+* GeoJSON <a name="geojson" />
+[GeoJSON](https://datatracker.ietf.org/doc/rfc7946/) is a JSON format for specifying geographic shape information.
+This is the default format for specifying plot geometries.
+
+* Scientific Filesystem <a name="scif" />
+We use the [Scientific Filesystem](https://sci-f.github.io/) to organize our applications, provide ease of execution, and to assist in reproducability.
+
+* Shapefile <a name="shapefile_def" />
+In this document we use the term "shapefile" to refer to all the files ending in `.shp`, `.shx`, `.dbf`, and `.prj` that have the same name.
+It can be used to specify geographic information and shapes associated with plot geometries.
+
+### Canopy Cover: Orthomosaic and plot boundaries <a name="can_om_shp" />
+The following steps are used to generate plot-level canopy cover values for a georeferenced orthomosaic image and plot boundaries using geographic information.
+We will first present the steps and then provide an example.
+
+1. Create a folder and copy the orthomosaic into it
+2. If using a [shapefile](#shapefile) or [GeoJSON](#geojson) file, copy those into the same folder as the orthomosaic image
+3. Create another folder for the output folders and files
+4. Run the docker container's `short_workflow` app specifying the name of the orthomosaic and either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries
+
+_NOTE_: that the orthomosaic must be the file name without any extensions; in other words, leave off the `.tif` when specifying it on the Docker command line.
+
+**For example:**
+
+In this example we're going to assume that the source image is named `orthomosaic.tif` and that we're using a shapefile named `plot_shapes.shp`.
```

Comment:
Would be useful to have sample data for sanity testing (either in the repository if it's small, or some way to fetch it from a data repository).

Reply:
I'll put together a GDrive package.

Reply:
Done